packages-native/animation/src/Animation.tsx

```diff
@@ -0,0 +1,77 @@
+import { Component, ReactNode, createElement } from "react";
+import { flattenStyles } from "@native-mobile-resources/util-widgets";
+import { Animation as AnimationType, View, Easing } from "react-native-animatable";
+
+import { AnimationProps } from "../typings/AnimationProps";
+import { defaultAnimationStyle, AnimationStyle } from "./ui/Styles";
+
+export type Props = AnimationProps<AnimationStyle>;
+type Direction = "normal" | "reverse" | "alternate" | "alternate-reverse";
+
+export class Animation extends Component<Props> {
```

Comment:
Why we are not using functional components ?

Reply:
Yes, could be. Template start with `Component` could re-write